Description: Avoid privacy breach in links to Github
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2018-09-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/docs/header.html
+++ b/docs/header.html
@@ -18,7 +18,7 @@
 <body>
 <div id="top"><!-- do not remove this div, it is closed by doxygen! -->
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 <div class="container">
--- a/html/admin.html
+++ b/html/admin.html
@@ -24,7 +24,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/audiobridgetest.html
+++ b/html/audiobridgetest.html
@@ -28,7 +28,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/citeus.html
+++ b/html/citeus.html
@@ -20,7 +20,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/demos.html
+++ b/html/demos.html
@@ -21,7 +21,7 @@
 </head>
 <body>
 	
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/devicetest.html
+++ b/html/devicetest.html
@@ -30,7 +30,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/docs/index.html
+++ b/html/docs/index.html
@@ -20,7 +20,7 @@
 </head>
 <body>
 	
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/echotest.html
+++ b/html/echotest.html
@@ -30,7 +30,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/index.html
+++ b/html/index.html
@@ -20,7 +20,7 @@
 </head>
 <body>
 	
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/nosiptest.html
+++ b/html/nosiptest.html
@@ -28,7 +28,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/recordplaytest.html
+++ b/html/recordplaytest.html
@@ -28,7 +28,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/screensharingtest.html
+++ b/html/screensharingtest.html
@@ -28,7 +28,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/sipretest.html
+++ b/html/sipretest.html
@@ -30,7 +30,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/siptest.html
+++ b/html/siptest.html
@@ -31,7 +31,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/streamingtest.html
+++ b/html/streamingtest.html
@@ -29,7 +29,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/support.html
+++ b/html/support.html
@@ -20,7 +20,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/textroomtest.html
+++ b/html/textroomtest.html
@@ -28,7 +28,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/videocalltest.html
+++ b/html/videocalltest.html
@@ -30,7 +30,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/videoroomtest.html
+++ b/html/videoroomtest.html
@@ -30,7 +30,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/voicemailtest.html
+++ b/html/voicemailtest.html
@@ -27,7 +27,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
--- a/html/vp9svctest.html
+++ b/html/vp9svctest.html
@@ -30,7 +30,7 @@
 </head>
 <body>
 
-<a href="https://github.com/meetecho/janus-gateway"><img style="position: absolute; top: 0; left: 0; border: 0; z-index: 1001;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png" alt="Fork me on GitHub"></a>
+<a href="https://github.com/meetecho/janus-gateway">Fork me on GitHub</a>
 
 <nav class="navbar navbar-default navbar-static-top">
 </nav>
